tools/clang-tidy-plugin/test/lit.cfg

```diff
@@ -10,11 +10,8 @@ config.test_exec_root = os.path.join(
 
 config.suffixes = ['.cpp', '.h']
 
-if config.cata_check_clang_tidy:
-    check_clang_tidy = config.cata_check_clang_tidy
-else:
-    check_clang_tidy = os.path.join(
-            config.cata_source_dir, "./tools/clang-tidy-plugin/test/check_clang_tidy.py" )
+check_clang_tidy = os.path.join(
+        config.cata_source_dir, "./tools/clang-tidy-plugin/test/check_clang_tidy.py" )
```

Comment:
This is needed on Windows to add custom executable path argument to the `check_clang_tidy.py` call.